guide/accessibility/responsive-content.md

```diff
@@ -0,0 +1,7 @@
+# Responsive content
```

Comment:
I'm not sold on the title cause I needed a verbal explanation on what it encompasses. However, it makes the most sense after the explanation